woocommerce/woocommerce-android
# [HACK Week: Wear App] Introduce Order Count and Visitors count complications

Summary
==========
Introduces the Today's Order Count and Today's Visitors Count complication on top of the Order Totals infrastructure introduced in https://github.com/woocommerce/woocommerce-android/pull/12511. 

This will allow users to follow a store's main three stats from the device's Watch face alone.

Screen Capture
==========
https://github.com/user-attachments/assets/8955a474-1bcd-4031-8e2b-6ba7b8f66776

How to Test
==========
1. In the Wear device, select a Watch face that supports Short Text complication types
2. Go to the WearOS Watch face editing menu and select the Order Count and Order Visitors complications
3. Make sure the data matches what is displayed in the store, but with a compact format.

Update release notes:


## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
